src/etl/presentation/api.py

```
@@ -34,6 +34,40 @@ class LoginRequest(BaseModel):
+    
+    try:
+        # Проверяем, отвечает ли Qdrant по сети. 
+        # У qdrant_client метод get_locks() — самый быстрый способ пинга.
+        if hasattr(repo, '_client'):
+            await repo._client.get_locks()
+            health_status["components"]["qdrant"] = "connected"
+        else:
+            # На случай, если клиент внутри репозитория называется иначе
+            health_status["components"]["qdrant"] = "connected (skipped deep check)"
+            
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)